test/e2e/test-apps/other/browser-profiling/src/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+  </head>
+  <body>
+    <script>
+      const crypto = require('crypto');
+      const {
+        addTracingExtensions,
+        init,
+        BrowserProfilingIntegration,
```

Comment:
We need both `RendererProfiling` and `BrowserProfilingIntegration`?

Reply:
`RendererProfiling` is only for the main node process and enables the header injection.